Ladybird/Android/src/main/cpp/LadybirdActivity.cpp

```diff
@@ -69,6 +71,11 @@ Java_org_serenityos_ladybird_LadybirdActivity_initNativeCode(JNIEnv* env, jobjec
     };
     Core::EventLoopManager::install(*event_loop_manager);
     s_main_event_loop = make<Core::EventLoop>();
+
+    // FIXME: It doesn't really make sense to have a WebView::Application, because we use Android services
+    // instead of separate processes.
+    // However, LibWebView depends on an Application object existing, so we create one here.
+    s_application = make<WebView::Application>(0, nullptr);
```

Comment:
Hmmm. at the very least we want an application to exist so that we can have process tracking in some kinda process widget, like the other chromes? But, we don't have one of those at the moment. Maybe Android/Android Studio's tooling is a lot better than anything we could cook up ourselves.

Reply:
That's valid, I've updated the comment to mention that we would want to track processes in the WebView::Application.

(and I suspect we can do something better than Android Studio's tooling, it's not very good at handling multiple processes...)